Solutions/Tanium/Workbooks/TaniumWorkbook.json

```diff
@@ -0,0 +1,529 @@
+{
+  "version": "Notebook/1.0",
+  "dependsOn": "[resourceId('Microsoft.Web/connections', parameters('ConnectionName'))]",
+  "items": [
+    {
+      "type": 11,
+      "content": {
+        "version": "LinkItem/1.0",
+        "style": "tabs",
+        "links": [
+          {
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Comply",
+            "subTarget": "Comply",
+            "preText": "Comply",
+            "style": "link"
+          },
+          {
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Patch",
+            "subTarget": "Patch",
+            "style": "link"
+          },
+          {
+            "cellValue": "selectedTab",
+            "linkTarget": "parameter",
+            "linkLabel": "Discover",
+            "subTarget": "Discover",
+            "style": "link"
+          }
+        ]
+      },
+      "name": "links - 8"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 1,
+            "content": {
+              "json": "## Top mission critical computers with CVSS hits >= 9.0"
+            },
+            "name": "text - 2"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "TaniumComplyVulnerabilities_CL \n| join TaniumMainAsset_CL on Computer_Name_s| where CVSS_Score_s startswith \"9\"\n| extend Host_0_HostName = Computer_Name_s\n| extend Malware_0_Name = CVE_s\n| extend IP_0_Address = IP_Address_s\n| summarize count() by Computer_Name_s | limit 4\n| sort by count_\n",
```

Comment:
@ian-hands-tanium, there are several tables used for workbook queries like TaniumComplyVulnerabilities_CL, TaniumComplyCompliance_CL,TaniumSentinelApplicablePatches_CL,TaniumSentinelApplicablePatches_CL,TaniumSentinelHighUptime_CL etc. Please add sample data for those too, thanks.

Reply:
@v-sabiraj sample data has been added. Sample data for TaniumSentinelApplicablePatches_CL was not added because it was removed our workflows for now.